modules/persistent-storage-csi-vsphere-limitations.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: CONCEPT
+[id="persistent-storage-csi-vsphere-limitations_{context}"]
+= vSphere CSI limitations
+
+The following limitations apply to the vSphere Container Storage Interface (CSI) Driver Operator:
+
+* The vSphere CSI Driver supports dynamic and static provisioning. However, when using static provisioning in the PV specifications, do not use the key `storage.kubernetes.io/csiProvisionerIdentity` in `csi.volumeAttributes` because this key indicates dynamically provisioned PVs.
```

Comment:
Where is this `csiProvisionerIdentity` text coming from?

Reply:
That is existing content. I just moved it into this new Limitations section.